index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Portfolio Page</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav id="toolbar">
+        <a href="#" id="logo">My Logo</a>
+        <div id="menu-bars-container">
+          <div id="menu-bars">
+            <div class="menu-bar"></div>
+            <div class="menu-bar"></div>
+            <div class="menu-bar"></div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="headline-container">
+      <h1 class="headline-header">
+        <span> I’m Hafed <br />Glad to see you! </span>
+      </h1>
+      <p class="headline-text">
+        I’m a software developer! I can help you build a product, feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a id="connect" href="#logo">Let's Connect</a>
+      <ul id="social-icons">
+        <li class="icon">
+          <a href="#" target="_blank">
+            <i class="fa-brands fa-twitter fa-lg"></i>
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://www.linkedin.com/in/hafedefhej/" target="_blank">
+            <i class="fa-brands fa-linkedin-in fa-lg"></i>
+          </a>
+        </li>
+        <li class="icon">
+          <a href="#" target="_blank">
+            <i class="fa-brands fa-medium fa-lg"></i>
+          </a>
+        </li>
+        <li class="icon">
+          <a href="https://github.com/hafedEfheij" target="_blank">
+            <i class="fa-brands fa-github fa-lg"></i>
+          </a>
+        </li>
+        <li class="icon">
+          <a href="#" target="_blank">
+            <i class="fa-brands fa-angellist fa-lg"></i>
+          </a>
+        </li>
+      </ul>
```

Comment:
Great job implementing some of the changes from the previous reviewer 👍. As requested, kindly style your page in accordance with the Figma design. 

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="376" alt="image" src="https://user-images.githubusercontent.com/73854999/205361275-b2ac3e05-85ca-4611-bebb-1b42672730c0.png">|<img width="376" alt="image" src="https://user-images.githubusercontent.com/73854999/205361325-155632a8-a274-4c79-86c6-6b5509c2aff3.png">|

The Icons are grey and not blue. Staying with the design's color, fonts, etc is essential.